extensions/kubectl-cli/.extfiles

```diff
@@ -0,0 +1,5 @@
+dist/**
+node_modules/**
+icon.png
+LICENSE
+package.json
```

Comment:
hello, what is it for ?

Reply:
This is just a config file with a list of files included into cdix archive. It is used in scripts/build.js.

Reply:
LICENSE file does not exists. I removed it form the list.